src/lib/viewers/box3d/image360/__tests__/Image360Controls-test.tsx

```diff
@@ -11,11 +10,10 @@ describe('lib/viewers/box3d/image360/Image360Controls', () => {
         onVrToggle: jest.fn(),
     });
 
-    const getWrapper = (props: Partial<Props>): ShallowWrapper =>
-        shallow(<Image360Controls {...getDefaults()} {...props} />);
+    const getWrapper = (props: Partial<Props>) => render(<Image360Controls {...getDefaults()} {...props} />);
```

Comment:
Do not use `wrapper` concept when working with React Testing Library. 

> _The name `wrapper` is old cruft from enzyme and we don't need that here. The return value from `render` is not "wrapping" anything. It's simply a collection of utilities that (thanks to the next thing) you should actually not often need anyway._

Source:[ Common mistakes with React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-wrapper-as-the-variable-name-for-the-return-value-from-render) by Kent C.Dodds

I highly recommend to get familiar with this article.